src/patterns/confirmation-pages/index.md

```diff
@@ -33,7 +59,15 @@ Your confirmation page must include:
 - a link to your [feedback page](https://www.gov.uk/service-manual/service-assessments/get-feedback-page)
 - a way for users to save a record of the transaction, for example, as a PDF
 
-{{ example({ group: "patterns", item: "confirmation-pages", example: "default", html: true, nunjucks: true, open: false, titleSuffix: "second", size: "xl" }) }}
+<div class="app-wcag-22" id="wcag-see-understand-text" role="note">
+  {{ govukTag({
+    text: "WCAG 2.2",
+    classes: "app-tag"
+  }) }}
+  <p>Do not include links, buttons or other interactive elements in the green confirmation panel, as the focus state will not have a minimum contrast ratio of 3:1 with the panel’s background colour.</p>
+</div>
+
+{{ example({group: "patterns", item: "confirmation-pages", example: "default", html: true, nunjucks: true, open: false, titleSuffix: "second", size: "xl"}) }}
```

Comment:
Taken out of WCAG 2.2 highlighting, kept message as standard guidance, saying that it's the Design System's focus style that does not meet minimum contrast.

```suggestion
Avoid including links, buttons or other interactive elements in the green confirmation panel, as the focus state will not meet the minimum contrast ratio of 3:1 with the panel’s background colour.
```